waspc/src/Wasp/AppSpec.hs

```diff
@@ -21,6 +44,50 @@ data AppSpec = AppSpec
     externalCodeDirPath :: !(Path' Abs (Dir ExternalCode.SourceExternalCodeDir)),
     -- | Absolute path to the directory in wasp project source that contains database migrations.
     migrationsDir :: Maybe (Path' Abs (Dir DbMigrationsDir)),
+    -- | Absolute path to the .env file in wasp project source. It contains env variables to be
+    -- provided to the server only during the development.
     dotEnvFile :: Maybe (Path' Abs File'),
+    -- | If true, it means project is being compiled for production/deployment -> it is being "built".
+    -- If false, it means project is being compiled for development purposes (e.g. "wasp start").
     isBuild :: Bool
   }
+
+-- TODO: Make this return "Named" declarations? I already have a comment for this somewhere else I think.
```

Comment:
I should probably create a github issue for this, or solve it now. `type` or `newtype`? Or nothing?

Reply:
I created an issue for this!